### PR TITLE
Added travis related content to enable checks

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: documentation/#64

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @nthomas-redhat @brainfunked @anmolbabu @nnDarshan please review.

**Comment by r0h4n** [comment]
- Update git commit msg to "tendrl-bug-id: Tendrl/documentation#64"  (similar to https://github.com/Tendrl/common/commit/8d821f571594026e8f99c737165da09d899a35e5)
- Explain what "redpen" is in the commit msg
- Why do we require tox and any python related bits in here?

**Review by r0h4n — CHANGES_REQUESTED** [review]
- Update git commit msg to "tendrl-bug-id: Tendrl/documentation#64"  (similar to https://github.com/Tendrl/common/commit/8d821f571594026e8f99c737165da09d899a35e5)
- Explain what "redpen" is in the commit msg
- Why do we require tox and any python related bits in here?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+before_install:
+  - "export TRAVIS_COMMIT_MSG=\\\"$(git log --format=%B --no-merges -n 1)\\"
+  - "pip install requests"
+git:
+  depth: 4
+env:
+- URL=https://github.com/redpen-cc/redpen/releases/download/redpen-1.7.6
+
+install:
+- "pip install tox-travis"
```
Comment (r0h4n) [inline .travis.yml]: tox-travis not required

File: check_commit_message.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/python
```
Comment (r0h4n) [inline check_commit_message.py:1]: This file is old, check current https://github.com/Tendrl/common/commit/8d821f571594026e8f99c737165da09d899a35e5

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (shtripat) [reply .travis.yml]: Done

File: check_commit_message.py (hunk 2 above)
Comment (shtripat) [reply check_commit_message.py:1]: Done. Will refresh this.

**Comment by shtripat** [comment]
The tools used here for validating the .adoc file (redpen) needs java-1.8. How do we make sure the travis env has the required java version.
It looks like "yum install java" in .tarvis.yml doesnt work.

@r0h4n ^^

**Comment by r0h4n** [comment]
@shtripat I am not aware of this tool, can we use another tool for validating .adoc which is more travis friendly

@brainfunked need your help here

**Pull request closed without merging**